cmd/skipper/main.go

```diff
@@ -57,6 +57,9 @@ const (
 	defaultTLSHandshakeTimeoutBackend = 60 * time.Second
 	defaultMaxIdleConnsBackend        = 0
 
+	// OAuth2:
+	defaultOAuthTokeninfoTimeout = 1500 * time.Millisecond // Not sure if this makes sense
```

Comment:
It makes sense, but still a bit opinionated in my view :smile: . How about time.Minute?

Reply:
I have changed it to `1.5 * time.Second`